docs/list-articles.html

```diff
@@ -26,200 +25,216 @@ <h1 id="auto-1">List of all the articles<span style="margin-left: 1em"></span></
       <hr></hr>
     </p>
     <p>
-      <a href="scheming.html">Scheming</a><font class="tmweb-entry-date">Wed Jan  6 18:18:54 2021 UTC</font>
+      <a href="articles-wishes.html">A wishlist for TeXmacs articles</a><font class="tmweb-entry-date">Fr 19 Feb 2021
+      14:25:32 GMT</font>
     </p>
     <div class="tmweb-entry-abstract">
-      We discuss some aspects of the choice of the Scheme language
-      implementation used in TeXmacs.
+      It would be useful to have some short articles or tutorials covering the
+      topics listed here below. This article collects them in the hope that
+      someone will contribute them to the blog. They can also be considered
+      interesting exercises to improve one own TeXmacs knowledge: learning by
+      teaching. In case you manage to cover, even partially, one of the topic,
+      please add a remark here to point to relevant material. If the topic is
+      covered, move it to the &ldquo;Already covered&rdquo; section.
     </div>
     <p>
-      <a href="list-articles.html">List of all the articles</a><font class="tmweb-entry-date">Wed Jan  6 17:39:23
-      2021 UTC</font>
+      <a href="list-articles.html">List of all the articles</a><font class="tmweb-entry-date">Fr 19 Feb 2021
```

Comment:
We have a problem: on your systems dates are generated in the German locale, while we need them in the English one. Exercise for you: how to modify the script to enforce the locale? (Also you can try to run the script in a shell with English and not German locale) :)

Reply:
> Exercise for you: how to modify the script to enforce the locale?

Will try and figure it out